### DISCUSSION TITLE
Displaying a .netframework window from a netcore window

### DISCUSSION THREAD

**Discussion by chasihle** [canton7/Stylet#237] [General]
hi,
I've written a simple report in SAP CrystalReports which is only supported under netframework. All of my other WPF forms are in a NetCore Project. I've added a reference to the netframework project and put the using clause in the core viewmodel. I'm using the the following code to display the window var viewModel = new InvoicePreviewViewModel(windowManager);
            this.windowManager.ShowDialog(viewModel);

this is being done in a UserControl and I pass windowmanager to the usercontrol.

when I attempt to display the window, I get the following error: StyletIoC.StyletIoCRegistrationException: 'No registrations found for service InvoicePreviewView.'

is this even possible or am I doing something wrong?  

thanks,
Charlie

**Comment by canton7** [comment]
Stylet is trying to find a type with the name `InvoicePreviewView`, and it's failing. If that type exists, then presumably the problem is that Stylet's looking in the wrong place.

This is *slightly* annoying to do at the moment: the easiest way to do it is probably:

```cs
public override void Configure()
{
    var viewManager = this.Container.Get<ViewManager>();
    viewManager.Assemblies.Add(assembly which contains your InvoicePreviewView);
}
```

I don't think this is anything to do with Framework vs Core, at least not from what you've said so far.

Does that make sense?
